code/modules/mining/abandoned_crates.dm

```diff
@@ -149,6 +149,234 @@
+					new /obj/item/storage/firstaid/regular(src)
+					new /obj/item/storage/toolbox/syndicate(src)
+				if("litestuff2")
+					new /obj/item/melee/classic_baton/police/telescopic(src)
+					new /obj/item/book/granter/spell/blind(src)
+					new /obj/item/storage/firstaid/toxin(src)
+					new /obj/item/grenade/clusterbuster/smoke(src)
+					new /obj/item/clothing/shoes/chameleon/noslip(src)
+				if("litestuff3")
+					new /obj/item/book/granter/spell/smoke(src)
+					new /obj/item/book/granter/spell/blind(src)
+					new /obj/item/clothing/under/chameleon(src)
+					new /obj/item/clothing/shoes/chameleon/noslip(src)
+				if("litestuff4")
+					new /obj/item/storage/firstaid/regular(src)
+					new /obj/item/storage/firstaid/toxin(src)
+					new /obj/item/storage/firstaid/brute(src)
+					new /obj/item/storage/firstaid/fire(src)
+					new /obj/item/storage/toolbox/syndicate(src)
+					new /obj/item/borg/upgrade/ddrill(src)
+					new /obj/item/borg/upgrade/soh(src)
+				if("litestuff5")
+					new /obj/item/grenade/plastic/c4(src)
+					new /obj/item/grenade/clusterbuster/smoke(src)
+					new /obj/item/storage/toolbox/syndicate(src)
+		if(117 to 120)
+			new /obj/item/tank/jetpack/void(src)
+			new /obj/item/clothing/mask/breath(src)
+			new /obj/item/clothing/head/helmet/space/nasavoid(src)
+			new /obj/item/clothing/suit/space/nasavoid(src)
+			new /obj/item/crowbar(src)
+			new /obj/item/stock_parts/cell(src)
+			new /obj/item/multitool(src)
+		if(121)
+			new /obj/item/clothing/gloves/chameleon/combat(src)
+			for(var/gloves in 6)
+				gloves = pick(subtypesof(/obj/item/clothing/gloves/color))
+				new gloves(src)
+		if(122)
+			new /obj/item/storage/box/syndie_kit/chameleon(src)
+		if(123)
+			new /obj/item/storage/box/syndie_kit/plasmachameleon(src)
+		if(124)
+			new /obj/item/radio/headset/chameleon/bowman(src)
+			new /obj/item/clothing/shoes/chameleon/noslip(src)
+			new /obj/item/reagent_containers/spray/waterflower(src)
+		if(125)
+			new /obj/item/clothing/shoes/clown_shoes/combat(src)
+			new /obj/item/storage/backpack/duffelbag/clown/syndie(src)
+			new /obj/item/pda/clown(src)
+			new /obj/item/megaphone/clown(src)
+			new /obj/item/clothing/mask/gas/clown_hat(src)
+		if(126)
+			new /obj/item/reagent_containers/spray/chemsprayer/janitor/clown(src)
+		if(127)
+			new /obj/item/clothing/head/crown/fancy(src)
+			new /obj/item/storage/box/monkeycubes(src)
+			new /obj/item/storage/box/monkeycubes(src)
+			new /obj/item/storage/box/monkeycubes(src)
+		if(128)
+			switch(pickweight(list("midstuff1" = 1, "midstuff2" = 1, "midstuff3" = 1, "midstuff4" = 1, "midstuff5" = 2, "midstuff6" = 1, "midstuff7" = 2, "midstuff8" = 1, "midstuff9" = 1)))
+				if("midstuff1")
+					new /obj/item/storage/firstaid/tactical(src)
+					new /obj/item/shield/riot/tele(src)
+					new /obj/item/pickaxe/drill/diamonddrill(src)
+				if("midstuff2")
+					new /obj/item/shield/energy(src)
+				if("midstuff3")
+					new /obj/item/dnainjector/lasereyesmut(src)
```

Comment:
This is an extremely powerful mutation that gives you a laser with unlimited ammo that previously only was attainable via the Gateway.